src/views/Home/hooks/useGetTopPoolsByApr.tsx

```diff
@@ -60,6 +62,7 @@ const useGetTopPoolsByApr = (isIntersecting: boolean) => {
     if (isIntersecting && fetchStatus === FetchStatus.Idle) {
       fetchPoolsPublicData()
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [dispatch, setFetchStatus, fetchStatus, topPools, isIntersecting])
```

Comment:
add `currentBlock` in deps

Reply:
it's intentional. Let's use `intitalBlock`

Reply:
Done